pkg/module/filterRule.go

```diff
@@ -10,4 +10,6 @@ const (
 	Include
```

Comment:
Would it make sense to rename it to `Public`?

Reply:
We can exclude everything - public as well as private modules. The difference between Include and Private is that in the first case we will fetch it from Olympus and not from private VCS but the name doesn't really reflect that.

Reply:
my concern is that you can explicitly say
```
+ github.com/a/b
```
which will Include this even though `github.com/a` was excluded. 
in this case, it is more intuitive to read it as include github.com/a/b then github.com/a/b is public
i understand this as an operation

Reply:
I understand your point but now we can say:
P github.com/a/b
which introduces the same problem.
So maybe IncludePrivate and IncludePublic?
Either way it's just an comment and can be resolved later.

Reply:
Yep and it is perfectly fine, `P github.com/a/b` treats github repo as a private, so it skips Olympus and goes to VCS directly. maybe we can rename Private to Direct so it reflects behavior.
i think this is also a valid scenario.
but these are the namings, the core feature behind those names implemented by Federico is good and we can start a discussion on this in a separate thread.